tmate/install.sh

```diff
@@ -34,21 +48,21 @@ sudo usermod -a -G dialout $UNAME
 
 
 #Check if we installed before...
-grep -q '617-440-8667' '/home/$UNAME/.bashrc'
+grep -q '617-440-8667' /home/$UNAME/.bashrc
```

Comment:
Why remove the quotes? If `$UNAME` contains a space, we would have a problem here

Reply:
single quotes there. it was preventing expansion of `$UNAME`